### PR TITLE
user guides: Review user documentation for Sending Messages

### PR DESCRIPTION
**Pull request by Ivche1337**

### CODE REVIEW

**Comment by Ivche1337** [comment]
https://codein.withgoogle.com/dashboard/task-instances/6045042567282688/

File: user-guides/ivche1337/Sending-messages.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   * Easy to follow
+   * No typos
+   * I could only follow the steps for linux and windows (I don't have mac)
+   * Adding screenshots will make it easier to follow for begginers
+
+## Reply-to-message
+   - Reply to a message by clicking
+   - Reply to a message through the message actions dropdown
+   - Reply to a message using the keyboard
+
```
Comment (digi0ps) [inline user-guides/ivche1337/Sending-messages.md]: Remove the blank line here.

File: user-guides/ivche1337/Sending-messages.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Sending-Messages
+
+## @-mention-a-team-member
+   * Easy to follow
+   * No typos
+   * Descriptive instructions
+   * /help/ screenshots match my dev instance behaviour
+
+
```
Comment (digi0ps) [inline user-guides/ivche1337/Sending-messages.md]: Remove the blank line here.

File: user-guides/ivche1337/Sending-messages.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Sending-Messages
```
Comment (digi0ps) [inline user-guides/ivche1337/Sending-messages.md]: You don't need the hyphen here, "Sending Messages" would be fine.

**Comment by digi0ps** [comment]
Hey @Ivche1337. 
Great work. I like the structure, the docs looks neat. 
I have commented on a few things, check them out.
Also, can you replace all the hyphens in the sub-headings with spaces? Spaces are much neater and consistent.

**Comment by Ivche1337** [comment]
Done!

**Comment by eeshangarg** [comment]
@Ivche1337: This looks great! One minor comment, please use `git commit --amend` to change the commit message to end with a period. Its a convention we like to enforce as much as possible! :) Thanks! :)

**Comment by Ivche1337** [comment]
Done!

**Comment by eeshangarg** [comment]
@Ivche1337: Great work! Merged! :)

**Pull request merged**